src/core/storage/secureStorage/secureStorage.ts

```diff
@@ -6,8 +6,11 @@ import {
 enum KeyStoreKeys {
   APP_PASSCODE = "app-login-passcode",
   APP_OP_PASSWORD = "app-operations-password",
+  PASSWORD_SKIPPED = "app-password-skip",
   SIGNIFY_BRAN = "signify-bran",
   MEERKAT_SEED = "app-meerkat-seed",
+  RECOVERY_WALLET = "recovery-wallet",
+  RECOVERY_WALLET_LAST_FAIL_ATTEMPT_TIME = "recovery-wallet-last-fail-attempt-time",
```

Comment:
I thought this was a future consideration in DTIS-898 @sdisalvo-crd. Tracking attempts like this is something the core team should handle, there are already tickets for this.

Reply:
I need to create a release for this week - maybe, it would be better to revert this commit and re-open the PR?

Reply:
Sure, @Sotatek-DukeVu would you mind please checking these notes from Fergal?

Reply:
PR for reverting is here: https://github.com/cardano-foundation/cf-identity-wallet/pull/524

Reply:
thanks Sal :)